### PR TITLE
Pipelines v2 content: KFP SDK

### PR DESCRIPTION
**Pull request by connor-mccarthy**
Initial pipelines v2 docs content, including only KFP SDK.

### CODE REVIEW

File: content/en/docs/components/pipelines/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+<!-- TODO: add UI screenshots for final pipeline -->
+In this tutorial we will deploy a KFP standalone deployment into an existing Kubernetes cluster, create and run a simple pipeline using the KFP SDK, view the pipeline on the KFP Dashboard, then create a more involved machine learning pipeline that uses additional KFP features.
+
+### 1) Deploy a KFP standalone instance into your cluster
+The first step is to deploy a KFP standalone instance into an existing Kubernetes cluster. If you don't yet have a Kubernetes cluster, see [Installation][installation]. For this step, you will need to have [kubectl](https://kubernetes.io/docs/tasks/tools/) installed.
+
+Once you have configured your [kubectl context](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to connect with your cluster. Replace `PIPELINE_VERSION` with the desired version of KFP and run the following script:
+
+```shell
+export PIPELINE_VERSION="2.0.0-alpha.3"
```
Comment (connor-mccarthy) [inline content/en/docs/components/pipelines/quickstart.md:17]: nit: alpha.4

**Review by chensun — APPROVED** [review]
/lgtm
/approve

Thanks @connor-mccarthy!

**Pull request merged**